include/swift/AST/DiagnosticsSema.def

```diff
@@ -6699,5 +6699,31 @@ ERROR(macro_undefined,PointsToFirstBadToken,
       "macro '%0' is undefined; use `-load-plugin-library` to specify dynamic "
       "libraries that contain this macro", (StringRef))
 
+//------------------------------------------------------------------------------
+// MARK: Move Only Errors
+//------------------------------------------------------------------------------
+
+ERROR(moveonly_copyable_type_with_moveonly_subtype, none,
+      "copyable %0 %1 can not contain a move only subtype. Found move only "
+      "subtype in %2 '%3.%4'",
+      (DescriptiveDeclKind, DeclName, DescriptiveDeclKind, StringRef, StringRef))
+ERROR(moveonly_copyable_type_with_moveonly_subtype2, none,
+      "copyable %0 %1 can not contain a move only type",
+      (DescriptiveDeclKind, DeclName))
+ERROR(add_nominal_moveonly_conformance,none,
+     "consider applying the '@_moveOnly' attribute %0 %1",
+     (DescriptiveDeclKind, DeclName))
+ERROR(non_moveonly_nominal,none,
+     "%0 %1 is not a move only type",
+      (DescriptiveDeclKind, DeclName))
+ERROR(moveonly_cannot_conform_to_protocol, none,
+     "Move only type %0 cannot conform yet to any protocols",
+     (DeclName))
+ERROR(moveonly_cannot_conform_to_protocol_with_name, none,
+     "Move only type %0 cannot conform to protocol %1. Move only types are "
+     "unable to conform to protocols yet",
```

Comment:
Diagnostic messages don't need to be pre-capitalized. I don't think we need the "unable to conform to protocols yet" bit, because even when we do add move-only protocols in the future, they will still be a new kind of thing, so it's likely to remain true that the move-only type cannot conform to the specific protocol being referred to.

Reply:
Ok